### PR TITLE
feat: improve troubleshooting section

### PR DESCRIPTION
**Pull request by benjaminParisel**
Based on work intiated in #83

### CODE REVIEW

**Comment by benjaminParisel** [comment]
what is missing:
- [ ]  icon just after anchor Troubleshooting
image

- [ ] if user put another title after troubleshooting-title. The icon is display too => We don't want it.. check if it's right

**Comment by benjaminParisel** [comment]
based on #88

File: src/stylesheets/globals/vars.scss [hunk 1]
```diff
@@ -48,6 +48,7 @@
   --color-card-border: #ebf2f2;
   --color-code-background: #f9f2f4;
   --color-code-font: #c7254e;
+  --color-troobleshooting-border: var(--color-admonition-important-bg);
```
Comment (jeromecambon) [inline src/stylesheets/globals/vars.scss:51]: duplicated var --color-troobleshooting-border. Remove this one.

File: src/stylesheets/globals/vars.scss [hunk 2]
```diff
@@ -227,12 +229,14 @@ html[data-theme='dark'] {
   --color-card-border: #9ba9c6;
   --color-code-background: #30334b;
   --color-code-font: #f7962e;
+  --color-troobleshooting-border: #f5c80071;
```
Comment (jeromecambon) [inline src/stylesheets/globals/vars.scss:232]: typo (tr**oo**bleshooting)

**Review by jeromecambon — CHANGES_REQUESTED** [review]

**Review by jeromecambon — APPROVED** [review]
LGTM!

**Pull request merged**